server/index.js

```diff
@@ -1,42 +0,0 @@
-const express = require ('express')
-const bodyParser = require ('body-parser')
-const cors = require('cors')
-
-const app = express()
-
-app.use(bodyParser.json())
-app.use(cors())
-
-let animalArr = [
-    {
-      name: 'Armadillo',
-      id: '1',
-      imageUrl: 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Nine-banded_Armadillo.jpg/1200px-Nine-banded_Armadillo.jpg'
-    },
-    {
-      name: 'Tiger',
-      id: '2',
-      imageUrl: 'http://kids.sandiegozoo.org/sites/default/files/2017-06/animal-hero-tiger_0.jpg'
-    }
-  ]
-
-  app.get('/api/animals', (req, res) => {
-      res.send(animalArr)
-      console.log('get request')   
-  })
-
-  app.post('/api/animals', (req, res) => {
-      animalArr.push(req.body)
-      console.log('animalArr', animalArr)
-      res.send(animalArr)
-      console.log(req.body)
-  })
-
-//   app.delete('/api/animals', (req, res) => {
-//       animalArr.pop()
-//       res.send(animalArr)
-//   })
```

Comment:
You're on the right track - for delete to work, you're going to want to use params, so your endpoint would look like this: 
```
app.delete('/api/animals/:id', (req, res, next) => {

})
```
In addition, popping is probably not the best way to delete, what I would do is I would filter through the animals array and include all ones that don't match the id you're trying to delete, so:
```
animalArr.filter(animal => animal.id !== req.params.id
```